src/pages/AboutPage.js

```diff
@@ -10,28 +10,18 @@ import fbLogo from '../assets/social_logos/Facebook_Logo.png';
 import igLogo from '../assets/social_logos/IG_Logo.png';
 import linkedinLogo from '../assets/social_logos/Linkedin_Logo.png';
 import twitterLogo from '../assets/social_logos/Twitter_Logo.png';
-import '../styling/AboutPage.css';
 
 const AboutPage = () => (
-  <Container className="container" fluid>
+  <Container className="about-page" fluid>
     <Row className="align-center">
-      <Col xs={12} md={6}>
+      <Col xs={12} md={4}>
         <Image src={logo} className="mx-auto d-block about-icon" fluid />
       </Col>
       <Col>
         <h1>Build for Black Lives</h1>
-        <h4>Building pro bono projects to support Black lives</h4>
-        <p>
-          Build for Black Lives, a platform created by HackBeanpot, connects individuals who have tech/design skills
-          with activists, organizers, and Black-owned businesses. We aim to provide pro bono work creating websites,
-          mobile apps, data visualizations, graphics, digital marketing, and/or any other technical projects the
-          organizations might need. The length of these projects could range from a few days to more long term.
-        </p>
+        <h3 className="about-subheader">Building pro bono projects to support Black lives</h3>
         <p>
-          HackBeanpot is a Boston-based hackathon, and as an organization we aim to build an inclusive community and
-          provide resources that allow our community to support each other in challenging times. Racial inequality has
-          long been an issue in the tech space. While change does not happen overnight, we want to make an active effort
-          to bring the tech community together to help support Black lives and fight for racial justice.
```

Comment:
I think we wanted to leave references to HackBeanpot in the About Us top section but just better explanation of how we tie into the site (there's revised copy in the copy doc that we can add later).

Reply:
lol yeah i started changing some of the copy to match the design then realized that all of the copy was probably gonna be changed anyway so i stopped